tests/sentry/templatetags/test_sentry_assets.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+from django.template import Context, Template
+from mock import Mock
+
+from sentry.testutils import TestCase
+
+
+class FeaturesTest(TestCase):
+    TEMPLATE = Template("""
+        {% load sentry_assets %}
+        {% locale_js_include request %}
+    """)
+
+    def test_supported_foreign_lang(self):
+        result = self.TEMPLATE.render(Context({
+            'request': Mock(LANGUAGE_CODE='fr'),  # French, in locale/catalogs.json
+        }))
+
+        assert '<script src="/_static/{version}/sentry/dist/moment/locale/fr.js" ></script>' in result
+
+    def test_unsupported_foreign_lang(self):
+        result = self.TEMPLATE.render(Context({
+            'request': Mock(LANGUAGE_CODE='ro'),  # Romanian, not in locale/catalogs.json
+        }))
+
+        assert '<script>' not in result
+
+    def test_english(self):
+        result = self.TEMPLATE.render(Context({
+            'request': Mock(LANGUAGE_CODE='en'),
+        }))
+
+        assert '<script>' not in result
+
+    def test_no_lang(self):
+        result = self.TEMPLATE.render(Context({
+            'request': Mock(),
+        }))
+
+        assert '<script>' not in result
```

Comment:
These tests seem useless, since `<script>` would never be in the output, even if we spit out a script tag. Since it'd be like, `'<script src="..."`. You could do:  `assert result.strip() == ''` since you more specifically want to make sure there is no output.